stdlib/messaging/kafka/src/main/ballerina/src/kafka/consumer.bal

```diff
@@ -260,23 +262,23 @@ public type Consumer client object {
         return consumerAssign(self, partitions);
     }
 
-# Closes consumer connection to the external Kafka broker.
-# ```ballerina
-# kafka:ConsumerError? result = consumer->close();
-# ```
-#
-# + duration - Timeout duration for the close operation execution
-# + return - `kafka:ConsumerError` if an error is encountered or else nil
+    # Closes consumer connection to the external Kafka broker.
```

Comment:
```suggestion
    # Closes the consumer connection of the external Kafka broker.
```

Reply:
These changes should've been reviewed in the Doc Hackathon. Same applies for all the existing API docs.